owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -308,10 +312,10 @@ class PassCodeActivity : AppCompatActivity() {
         explanationVisibility: Int
     ) {
         Arrays.fill(passCodeDigits, null)
-        passCodeError.setText(errorMessage)
-        passCodeError.visibility = View.VISIBLE
-        passCodeHdr.text = headerMessage // TODO check if really needed
-        passCodeHdrExplanation.visibility = explanationVisibility // TODO check if really needed
+        binding.error.setText(errorMessage)
+        binding.error.visibility = View.VISIBLE
+        binding.header.text = headerMessage // TODO check if really needed
+        binding.explanation.visibility = explanationVisibility // TODO check if really needed
```

Comment:
Apparently, they are really needed, right? We could remove the comments just in case 👍

Reply:
👍